ioctl/cmd/node/nodedelegate.go

```diff
@@ -365,3 +365,16 @@ func getAllStakingCandidates(chainClient iotexapi.APIServiceClient) (candidateLi
 	}
 	return
 }
+
+func getCandidateRewardAddressByName(cli iotexapi.APIServiceClient, name string) (string, bool, error) {
```

Comment:
1. `getCandidateRewardAddressByAddressOrName() (string, error)`
2. move `address, err := util.Address(arg)` to here